documentation/resources/dashboard-folder-assignment.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just a preview of the image that's located in this `.svb`
![image](https://user-images.githubusercontent.com/35736504/91549724-44750880-e91f-11ea-9d14-a035a7ac2ee1.png)